x-pack/test/security_solution_cypress/cypress/e2e/investigations/alerts/detection_page_filters.cy.ts

```diff
@@ -117,7 +116,7 @@ describe.skip(`Detections : Page Filters`, { tags: ['@ess', '@serverless'] }, ()
     waitForAlerts();
   });
 
-  it('Default page filters are populated when nothing is provided in the URL', () => {
+  it('should populate page filters with default values when nothing is provided in the URL', () => {
     assertFilterControlsWithFilterObject();
   });
 
```

Comment:
line 125 I'm wondering why in this test we have to `waitForPageFilters()` but the other ones don't have to?

Reply:
also line 131, I'm not sure I see why we would need to call the `discardFilterGroupControls()` method at the end of the test?

Reply:
and line 161, looking at the implementation of the `deleteFilterGroupControl`, we might be able to remove `cy.get(FILTER_GROUP_CONTROL_CONFIRM_DIALOG).should('be.visible');`. I say might because maybe this is a scenario where we need some transition to finish or something...

Reply:
all Done ✅